python/CMakeLists.txt

```diff
@@ -826,8 +872,7 @@ foreach(module ${CYTHON_EXTENSIONS})
     set_target_properties(${module_name} PROPERTIES COMPILE_DEFINITIONS
                                                     "CYTHON_TRACE=1;CYTHON_TRACE_NOGIL=1")
   endif()
-
-  target_link_libraries(${module_name} PRIVATE ${LINK_LIBS})
+  target_link_libraries(${module_name} PUBLIC ${LINK_LIBS})
```

Comment:
Why do we need this change?
I think that Cython modules aren't linked to others.